### PR TITLE
Remove unwanted files from enforcer

### PR DESCRIPTION
**Pull request by praminda**
### Purpose

Enforcer was packaing some unwanted files into it's docker image and this PR will remove those files.
### Issues

Task

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

standalone docker-compose

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: enforcer-parent/enforcer/src/main/resources/Dockerfile [hunk 1]
```diff
@@ -66,7 +66,7 @@ RUN chmod +x ${GRPC_HEALTH_PROBE_PATH}
 WORKDIR ${MG_USER_HOME}
 USER ${MG_USER}
 
-COPY maven/ lib/
+COPY maven/lib/ lib/
```
Comment (renuka-fernando) [inline enforcer-parent/enforcer/src/main/resources/Dockerfile:69]: Shall we do the same for Dockerfile.ubuntu

Comment (praminda) [reply enforcer-parent/enforcer/src/main/resources/Dockerfile:69]: missed it. added now

**Review by renuka-fernando — APPROVED** [review]

**Review by pubudu538 — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211215.32

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211215.43

**Pull request merged**